# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kterm/glibc-2.3.4.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Some invalid assumptions that fail since glibc-2.3.4.

  - Rene Rebe <rene@exactcode.de>

--- kterm-6.2.0/button.c.vanilla	2005-01-06 19:13:16.034016512 +0100
+++ kterm-6.2.0/button.c	2005-01-06 19:13:21.214229000 +0100
@@ -41,8 +41,6 @@
 #include "error.h"
 #include "menu.h"
 
-extern char *malloc();
-
 extern void DoSecureKeyboard();
 
 #define KeyState(x) (((x) & (ShiftMask|ControlMask)) + (((x) & Mod1Mask) ? 2 : 0))
--- kterm-6.2.0/charproc.c.vanilla	2005-01-06 19:13:40.348320176 +0100
+++ kterm-6.2.0/charproc.c	2005-01-06 19:13:52.982399504 +0100
@@ -101,9 +101,6 @@
 
 extern XtAppContext app_con;
 extern Widget toplevel;
-extern void exit();
-extern char *malloc();
-extern char *realloc();
 extern fd_set Select_mask;
 extern fd_set X_mask;
 extern fd_set pty_mask;
